### PR TITLE
Don't persist perftest data downloads on TeamCity…

### PR DESCRIPTION
**Pull request by bspratt**
…, as they won't actually get re-used and may take up so much room as to interfere with other builds

### CODE REVIEW

**Comment by brendanx67** [comment]
Curious whether we should also stop leaving the tutorial/perf test ZIP files in the Downloads directory. Does that help performance at all run-to-run, or do we end up tearing down the VM and starting from scratch again anyway? Should we simply be downloading these ZIP files from S3 every time? If we deleted the ZIP files after every test, that would save even more disk space over the life of a Tutorial/Perf test run.

**Comment by brendanx67** [comment]
If the persistent extracted files won't get automatically reused, then it would seem to me that neither would the ZIP files they came from. But, I don't see the code removing those.

File: pwiz_tools/Skyline/TestUtil/AbstractUnitTest.cs [hunk 1]
```diff
@@ -219,12 +220,14 @@ public string[] TestFilesZipPaths
             }
         }
 
+        public static bool DownloadFromS3 => Environment.GetEnvironmentVariable(@"SKYLINE_DOWNLOAD_FROM_S3") == "1";
```
[the remark this answers is not in the document]
Comment (bspratt) [reply pwiz_tools/Skyline/TestUtil/AbstractUnitTest.cs:223]: Fair enough.

[the remark this answers is not in the document]
Comment (bspratt) [reply pwiz_tools/Skyline/TestUtil/AbstractUnitTest.cs]: Now deleting tutorial data downloads too.

**Pull request merged**